cartridges/int_bolt_embedded_sfra/cartridge/scripts/services/account.js

```diff
@@ -174,7 +174,8 @@ function addPaymentMethodInfoToBasket(basket, boltPaymentMethods) {
     }
     // store all payment methods in the basket so that it can be later chosen by the customer
     Transaction.wrap(function () {
-        basket.getCustom().boltPaymentMethods = JSON.stringify(boltPaymentMethods);
+        // The maximum byte length of basket.custom property is 4000, so we only get the first 5 payment methods to avoid string value truncated
+        basket.getCustom().boltPaymentMethods = JSON.stringify(boltPaymentMethods.length > 5 ? boltPaymentMethods.slice(0, 5) : boltPaymentMethods);
```

Comment:
I have changed the type for boltPaymentMethods from string to text in this [PR](https://github.com/BoltApp/bolt-demandware-embedded/pull/161/files) last month so it should be good now. You might need to reimport the meta.

Reply:
Well, it works after importing the latest version of meta.